### PR TITLE
SorrTask1039 Make generate_limit_order_price() private

### PR DESCRIPTION
**Pull request by smitpatel49**
Task #1039

### CODE REVIEW

File: research_amp/cc/notebooks/CMTask3352_pegged_at_mid_algo_simulation.py
Comment (DanilYachmenev) [inline research_amp/cc/notebooks/CMTask3352_pegged_at_mid_algo_simulation.py:1]: We do not update notebooks that do not start from `Master_...`
iow we update only master notebooks, others are just kept as they are

could you pls revert notebook to its master state?

**Review by DanilYachmenev — COMMENTED** [review]
1 nit and lgtm

**Review by DanilYachmenev — COMMENTED** [review]
lgtm

**Review by gpsaggese — COMMENTED** [review]
LG

**Review by DanilYachmenev — APPROVED** [review]

**Pull request merged**